### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by BeeHiveJava**

### CODE REVIEW

File: web/src/main/java/nl/tehdreamteam/se42/web/soap/SoapWebService.java [hunk 1]
```diff
@@ -15,6 +17,8 @@
  */
 public class SoapWebService implements Service {
 
+    private final Logger logger = LogManager.getLogger(getClass().getSimpleName());
```
Comment (TehMessiah) [inline web/src/main/java/nl/tehdreamteam/se42/web/soap/SoapWebService.java]: MAJOR Make the "logger" logger private static final. [rule](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1312)

File: web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserController.java [hunk 2]
```diff
@@ -11,8 +13,12 @@
 @WebService
 public class SoapUserController implements UserController {
 
+    private final Logger logger = LogManager.getLogger(getClass().getSimpleName());
```
Comment (TehMessiah) [inline web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserController.java]: MAJOR Make the "logger" logger private static final. [rule](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS1312)

File: web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserController.java [hunk 3]
```diff
@@ -11,8 +13,12 @@
...
+
     @Override
     public String login(@WebParam(name = "username") String username, @WebParam(name = "password") String password) {
+        logger.debug("Soap request for login (username='{}', password='{}').", username, password);
```
Comment (TehMessiah) [inline web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserController.java]: BLOCKER Remove this hard-coded password. [rule](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS2068)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 3 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserController.java [hunk 4]
```diff
@@ -11,8 +13,12 @@
 @WebService
 public class SoapUserController implements UserController {
 
+    private static final Logger logger = LogManager.getLogger(SoapUserController.class.getSimpleName());
+
     @Override
     public String login(@WebParam(name = "username") String username, @WebParam(name = "password") String password) {
+        logger.debug("Soap request for login (username='{}', password='{}').", username, password);
```
Comment (ghost) [inline web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserController.java:20]: BLOCKER Remove this hard-coded password. [rule](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS2068)

**Comment by ghost** [comment]
SonarQube analysis reported 1 issue
* ![BLOCKER][BLOCKER] 1 blocker

Watch the comments in this conversation to review them.

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'

File: web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserController.java (hunk 4 above)
Comment (BeeHiveJava) [reply web/src/main/java/nl/tehdreamteam/se42/web/soap/user/SoapUserController.java:20]: Not hardcoded.

**Pull request merged**